tutorials/katacoda/1-serving-intro/step3.md

```diff
@@ -1,47 +1,63 @@
-## Blue/Green deploy
-The Knative `service` resource creates additional resources "route, configuration and revision" to manage the lifecycle of the application.
-- revision: just like a git revision, any change to the Service's `spec.template` results in a new revision
-- route: control traffic to several revisions
+## Scaling to Zero
 
-You can list those resources by running `kubectl get ksvc,configuration,route,revision` or by using the `kn` cli
+**Remember those super powers 🚀 we talked about?** One of Knative Serving's powers is built-in automatic scaling (autoscaling).
+This means your Knative Service only spins up your application to perform its job -- in this case, saying "Hello world!"
+-- if it is needed; otherwise, it will "scale to zero" by spinning down and waiting for a new request to come in.
 
-We will now update the service to change the `TARGET` env variable to `green`.
+> **What about scaling up to meet increased demand?**
+> Knative Autoscaling also allows you to easily configure your service to scale up (horizontal autoscaling) to meet 
+> increased demand as well as control the number of instances that spin up using
+> [concurrency limits and other options](https://knative.dev/docs/serving/autoscaling/concurrency/),
+> but that's beyond the scope of this tutorial.
 
-But, before we do that, let us update the revision name to "hello-v1", so that we can direct traffic to it.
-```
-kn service update demo --revision-name="demo-v1"```{{execute T1}}
 
-Now, let's update the env variable to `green`, but let's do it as a dark launch i.e. zero traffic will go to this new revision:
-```
-kn service update demo --env TARGET=green --revision-name="demo-v2" --traffic demo-v1=100,demo-v2=0
+**Let's see this in action!** We're going to peek under the hood at the [Pod](https://kubernetes.io/docs/concepts/workloads/pods/)
+in Kubernetes where our Knative Service is
+running to watch our "Hello world!" Service scale up and down.
+
+### Run your Knative Service
+Let's run our "Hello world!" Service just one more time. Use your terminal with curl.
+```sh
+curl -H "Host: hello.default.example.com" $externalIP
 ```{{execute T1}}
 
-This will result in a new `revision` being created. Verify this by running `kn revision list`{{execute T1}}.
+You can watch the pods and see how they scale to zero after traffic stops going to the URL (opened in a new Terminal tab,
+if the command does not run automatically, click the command bellow again)
```

Comment:
I am trying to describe this. Before you get to this point you are seeing this:
<img width="1101" alt="Screen Shot 2022-04-29 at 1 10 50 PM" src="https://user-images.githubusercontent.com/101346415/166000092-a830c6b6-fbf5-4955-a135-41910375cb9c.png">

Then, we click the command `kubectl get pod -l serving.knative.dev/service=hello -w`
You now see this:
<img width="1272" alt="Screen Shot 2022-04-29 at 1 11 18 PM" src="https://user-images.githubusercontent.com/101346415/166000180-d12d4506-e752-4594-9418-47189e6e525a.png">

When you click it it will open a new terminal tab.

Is there a better way to describe this ?
thanks

Reply:
OK, I think I understand what you mean. Would this make sense: 
> You can watch the pods and see how they scale to zero after traffic stops going to the URL by clicking the following command. This opens in a new terminal tab.
> `kubectl get pod -l serving.knative.dev/service=hello -w`
> If the command does not run automatically, click the command again.

So do users just need to click these commands, instead of copy, pasting, and running them?

Reply:
Correct!. Just click it. No need to copy pasting it. Pretty cool 😎